src/commands/run.ts

```diff
@@ -89,7 +103,10 @@ export default class Run extends BaseExecutionCommand {
       component => new ColorizedWritable(component.name, maxPrefixLength)
     )
 
-    const executorService = new ParallelProcessExecutorService(executionOptions)
+    const executorService = new ParallelProcessExecutorService(
+      executionOptions,
+      10
```

Comment:
Didn't we say to put this equal to the number of components?

Reply:
@zonia3000 I'll commit that change in the next PR (--all)